tests/contract-playground/src/ContractLocksEther.sol

```diff
@@ -0,0 +1,137 @@
+// SPDX-License-Identifier: MIT
```

Comment:
This is a great test file that covers inheritance, but I don't see any reference to eth `send()` and `transfer()` functions that could also act as a withdraw function. Could you please add cases for these?

Reply:
✅